setup.py: remove pytest-runner requirement

We don't need to run the tests to install the package anyhow.

Signed-off-by: Frederik Aalund <fpa@sbtinstruments.com>

--- a/setup.py	2020-01-29 09:35:06.823841696 +0100
+++ b/setup.py	2020-01-29 09:35:21.303841696 +0100
@@ -50,7 +50,6 @@ setup(
         'Programming Language :: Python :: Implementation :: CPython',
         'Topic :: Software Development :: Libraries :: Python Modules'
     ],
-    setup_requires=['pytest-runner'],
     tests_require=['pytest>=4.6.1', 'pytest-asyncio>=0.10.0', 'coverage>=4.5.3'],
 
     package_data={'rx': ['py.typed']},

